packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -91,27 +91,38 @@ async function setupResourceServerSession(
   // authenticated request to any actual resource (even public ones) does,
   // so we fetch the user's WebID before checking the /session endpoint.
   await authenticatedFetch(webId);
-  const resourceServerResponse = await authenticatedFetch(
-    `${resourceServerIri}/session`
-  );
-
-  if (resourceServerResponse.status === 200) {
-    await storageUtility.storeResourceServerSessionInfo(
-      webId,
-      resourceServerIri,
-      // Note that here, if the lifespan of the cookie was returned by the server,
-      // we'd expect a relative value (the remaining time of validity) rather than
-      // an absolute one (the moment when the cookie expires).
-      Date.now() + DEFAULT_LIFESPAN
+  try {
+    const resourceServerResponse = await authenticatedFetch(
+      `${resourceServerIri}/session`
```

Comment:
Jeez - seeing these hard-coded magic string values buried deep inside code like this makes the Java-developer in me squirm like crazy. I really think we should get these out - what's the issue with just moving these to a 'constant.ts' file or something (ideally into a vocab, but a local 'constant.ts' file seems like an easy first baby step)...?

Reply:
This is intentionally self-contained so that it is easy to remove, because it is meant to be a temporary solution.

Reply:
Ah Ok - well, then a big dirty `TODO:` comment to that effect is fine instead I suppose...